Source/Urho3D/CSharp/Swig/Urho3D.i

```diff
@@ -733,18 +733,19 @@ public:
 
 //// --------------------------------------- Physics ---------------------------------------
 #if defined(URHO3D_PHYSICS)
-//%ignore Urho3D::TriangleMeshData::meshInterface_;
-//%ignore Urho3D::TriangleMeshData::shape_;
-//%ignore Urho3D::TriangleMeshData::infoMap_;
-//%ignore Urho3D::GImpactMeshData::meshInterface_;
-//
-//%include "generated/Urho3D/_pre_physics.i"
-//%include "Urho3D/Physics/CollisionShape.h"
-//%include "Urho3D/Physics/Constraint.h"
-//%include "Urho3D/Physics/PhysicsWorld.h"
-//%include "Urho3D/Physics/RaycastVehicle.h"
-//%include "Urho3D/Physics/RigidBody.h"
-//
+%ignore Urho3D::TriangleMeshData::meshInterface_;
+%ignore Urho3D::TriangleMeshData::shape_;
+%ignore Urho3D::TriangleMeshData::infoMap_;
+%ignore Urho3D::GImpactMeshData::meshInterface_;
+
+%include "generated/Urho3D/_pre_physics.i"
+%include "Urho3D/Physics/CollisionShape.h"
+%include "Urho3D/Physics/Constraint.h"
+%include "Urho3D/Physics/PhysicsWorld.h"
+%include "Urho3D/Physics/RaycastVehicle.h"
+%include "Urho3D/Physics/RigidBody.h"
+%include "Urho3D/Physics/KinematicCharacterController.h"
```

Comment:
Does uncommenting these lines produce more `SWIGTYPE_` classes when wrapper is generated? Ideally there should be none, but i let that slide for too long already. We should at least try to not increase this number.